src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -617,7 +609,7 @@ function MoneyRequestReportPreviewContent({
                                             isLoading={!isOffline && !canAllowSettlement}
                                         />
                                     )}
-                                    {!!shouldShowExportIntegrationButton && !shouldShowSettlementButton && !shouldShowRBR && (
+                                    {buttonType === IOU_REPORT_PREVIEW_BUTTON.EXPORT && !!connectedIntegration && (
```

Comment:
This is not needed here right? and if it is now, we should remove the dependency so the logic is already added to the method that returns the `buttonType`
```suggestion
                                    {buttonType === IOU_REPORT_PREVIEW_BUTTON.EXPORT && (
```

Reply:
@JakubKorytko I still do not understand this, we already have the variable in here:

 const shouldShowExportIntegrationButton = !shouldShowPayButton && !shouldShowSubmitButton && !!connectedIntegration && isAdmin && canBeExported(iouReport);

that is passed to the `getIOUReportPreviewButtonType` so the buttonType === IOU_REPORT_PREVIEW_BUTTON.EXPORT if an only if `!!connectedIntegration` is true